msi-ec.c

```diff
@@ -58,9 +62,9 @@ static int ec_read_seq(u8 addr, u8 *buf, int len)
 // ============================================================ //
 
 static ssize_t charge_control_threshold_show(u8 offset,
-				struct device *device,
-				struct device_attribute *attr,
-				char *buf)
+											 struct device *device,
```

Comment:
I am not sure about if it's correct, but I was checking another kernel modules and the function args only have 8 white-spaces. I am unable to find further information